localization/react-intl/src/app/components/team/SmoochBot/SmoochBotMainMenuOption.json

```diff
@@ -2,12 +2,12 @@
   {
     "id": "smoochBotMainMenuOption.scenario",
     "description": "Title of dialog that opens to add a new option to tipline bot main menu",
-    "defaultMessage": "Scenario"
+    "defaultMessage": "Menu Option"
   },
   {
     "id": "smoochBotMainMenuOption.label",
     "description": "Text field label on dialog that opens to add a new option to tipline bot main menu",
-    "defaultMessage": "Button text - 24 characters limit"
+    "defaultMessage": "Button text"
```

Comment:
If I'm not mistaken, this is a restriction imposed by WhatsApp. Has it been dropped? @caiosba can you advise, please?

Reply:
The restriction is still in there @amoedoamorim 
Since per @dariusk comment, I switched this to a limited text area, the information about the character limit is moved to the helper content in the component

Reply:
Oh, ok! That's my bad, thanks!